Doppler.Sap/Models/DopplerUserDTO.cs

```diff
@@ -35,5 +30,7 @@ public class DopplerUserDto
         public string FederalTaxType { get; set; }
         public string FederalTaxID { get; set; }
         public int BillingSystemId { get; set; }
+        public int ClientManagerType { get; set; }
+        public bool IsFromRelay { get; set; }
```

Comment:
What do you think about using a string convention or an enum in place of a boolean?

```suggestion
        public string Product { get; set; } // `relay` or `doppler`
```

Reply:
I had thought of using a string but currently exists another bool property for ClientManager and for it reason I created a bool property for Relay and then I assumed if IsClientManager == false and IsFromRelay == false  is from Doppler.

But I don't have any problem to replace the bool property to string.

Reply:
I talked with Andres and for now is ok but I will create a new task to replace both properties:
```
public bool IsClientManager { get; set; }
public bool IsFromRelay { get; set; }
```

By:
````
public string Product { get; set; }